addons/mail/tests/test_res_partner.py

```diff
@@ -481,6 +481,14 @@ def test_message_get_suggested_recipients(self):
                     'partner_id': partner.id,
                 }])
 
+    @users('admin')
+    def test_mention_cross_company_portal_user(self):
```

Comment:
There is already a ``test_discuss_mention_suggestions_priority`` which can be improved probably instead of adding another similar test